game.js

```diff
@@ -0,0 +1,90 @@
+class Game {
+  constructor() {
+    this.frame = 1;
+    this.roll = 1;
+    this.pinsRolled = [[], [], [], [], [], [], [], [], [], []];
+    this.continue = true;
+    this.score = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+  };
+
+  getRoll() {
+    return this.roll;
+  };
+
+  getFrame() {
+    return this.frame;
+  };
+
+  getPinsRolled() {
+    return this.pinsRolled;
+  };
+
+  getContinue() {
+    return this.continue;
+  };
+
+  checkIfConotinue() {
+    if (this.frame === 10 && this.roll === 2 && this.pinsRolled[9].reduce((partialSum, now) => partialSum + now) < 10) {
+      this.continue = false;
+    };
+    if (this.frame === 10 && this.roll === 3) {
+      this.continue = false;
+    }
+  }
+
+  rollPin(pinNum) {
+    this.pinsRolled[this.frame - 1].push(pinNum);
+    // writeToScoreboard starts
+    this.writeToScoreboard(pinNum);
+    // writeToScoreboard ends
+    
+    this.checkIfConotinue();
```

Comment:
Good use of conditions as it stops code from returning wrong output if a user tries to enter wrong input